data-model/config/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+        "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<suppressions>
+    <suppress checks="FileLength" files="SqlQueryBuilder.java" />
```

Comment:
I have added this to the main checkstyle config but it is being overwritten with sync files job. It works here and it fits better here.